draft-ietf-quic-ack-frequency.md

```diff
@@ -365,6 +365,16 @@ reducing the ACK rate compared to {{Section 13.2.1 of QUIC-TRANSPORT}} during
 extreme congestion or when peers are using DCTCP {{?RFC8257}} or other
 congestion controllers that mark more frequently than classic ECN {{?RFC3168}}.
 
+If the most recent ACK_FREQUENCY frame an endpoint has received from the peer
+has an `Ignore CE` value of `true` (0x01), receipt of a CE marked packet
+SHOULD NOT cause an endpoint to send an immediate acknowledgement.  The endpoint
+still sends an immediate acknowledgement if it would have for a non CE marked
+packet.  Otherwise, the CE marks are reported in the next acknowledgement.
```

Comment:
`Otherwise` doesn't read well here. I don't know what would work better, probably "And"

Reply:
Agreed, reworded.